config/config.h.in

```diff
@@ -61,6 +60,7 @@
 #define TIME_WITH_SYS_TIME 1
 #cmakedefine UNIX
 #cmakedefine USE_FNAME_CASE
+#cmakedefine MSVC
```

Comment:
We do but WIN32 is already set in headers.

On second thought we should just use _MSC_VER instead since that is the standard. I'm dropping this commit.